src/common/config_opts.h

```diff
@@ -1317,6 +1317,9 @@ OPTION(rgw_realm_reconfigure_delay, OPT_DOUBLE, 2) // seconds to wait before rel
 OPTION(rgw_period_push_interval, OPT_DOUBLE, 2) // seconds to wait before retrying "period push"
 OPTION(rgw_period_push_interval_max, OPT_DOUBLE, 30) // maximum interval after exponential backoff
 
+OPTION(rgw_compression_enabled, OPT_BOOL, true) // to use compression on rgw level
```

Comment:
I think we don't need this boolean. Instead ```rgw_compression_type``` could be examined for some special value - empty string seems to be a good candidate.